### DISCUSSION TITLE
Stream Raspberry cam and show live image on HDMI

### DISCUSSION THREAD

**Discussion by cg089** [bluenviron/mediamtx#3520] [Questions]
### Question

I use a Raspberry as a door camera. For this purpose, I have installed a mini Raspberry camera in the peephole, the Raspberry itself hangs on the door and has a small 3-inch display permanently connected to it. This way I can see who is standing in front of the door live and I can also stream the live image. So far I have used Live555 streaming server for this whic outputs a live image on HDMI the moment it starts.

Because of performance issues I would like to try MediaMTX. Is it possible to stream the live image via RTSP and output it to the HDMI output at the same time? What do I have to set for this?

**Comment by aler9** [comment]
Hello, you can achieve this by using VLC to read the RPI camera stream, provided by MediaMTX, through RTSP:

```
vlc --network-caching=50 rtsp://localhost:8554/cam
```

This approach works with both console and desktop, and has the advantage that it's compatible with any stream coming from the network, not just with the stream of the onboard RPI camera.

Some years ago there was omxplayer, but all its features have been moved to VLC and it has been deprecated.

Regarding native support, MediaMTX has been conceived as a headless server, and adding support for displaying streams on screen would be out of scope (and compatible with the RPI only and not any other platform, since they use two different APIs).

[a comment by an automated account is not shown]
